### ISSUE TITLE
Everything is wrong.

### ISSUE THREAD

**Issue by theSteveMitchell** [theSteveMitchell/countable_infinty#1]

**Comment by cameron-martin** [comment]
> There is no real number b that does not correspond to f(n) for some value of n

Yes there is - any non-terminating decimal. You've just proved that the set of terminating decimals are countable, which is a subset of the rationals.

**Comment by theSteveMitchell** [comment]
@cameron-martin yes you are correct.
